topic_monitor/setup.py

```diff
@@ -1,17 +1,27 @@
-from ament_python.data_files import get_data_files
-from ament_python.script_dir import install_scripts_to_libexec
+import sys
+
 from setuptools import find_packages
 from setuptools import setup
 
 package_name = 'topic_monitor'
-data_files = get_data_files(package_name)
-install_scripts_to_libexec(package_name)
+
+# install scripts into libexec path
+if 'develop' in sys.argv:
+    sys.argv[sys.argv.index('develop') + 1:sys.argv.index('develop') + 1] = \
```

Comment:
Same question as above (link to be edited in pending submission). How does this replace a manually specified script-dir argument?